install/helm/README.md

```diff
@@ -0,0 +1,64 @@
+### Open Match Helm Chart Templates
+This directory contains the [helm](https://helm.sh/ "helm") chart templates used to customize and deploy Open Match at ease.
+
+Templates under the `templates/` directory are for the core components in Open Match - e.g. backend, frontend, mmlogic, synchronizor, some security policies, and configmaps are defined under this folder.
+
+Open Match also provides templates for optional components that are disabled by default under the `subcharts/` directory.
+1. `open-match-demo` contains the chart definitions of an example director.
+2. `open-match-customize` contains flexible templates to deploy your own matchfunction and evaluator.
+3. `open-match-telemetry` contains monitoring supports for Open Match, you may choose to enable/disable [jaeger](https://www.jaegertracing.io/ "jaeger"), [prometheus](http://prometheus.io "prometheus"), [stackdriver](https://cloud.google.com/stackdriver/ "stackdriver"), [zipkin](https://zipkin.io/ "zipkin"), and [grafana](https://grafana.com/ "grafana") by overriding the config values in the provided templates.
+4. `open-match-test` contains templates of the end-to-end in-cluster tests and distributed stress tests for Open Match.
+
+You may control the behavior of Open Match by overriding the configs in `install/helm/open-match/values.yaml` file. Here are a few examples:
+
+```diff
+# install/helm/open-match/values.yaml
+
+# Setup Open Match with customized ticket indices
+ticketIndices:
++ - myfirstindice
++ - mysecondice
++ - ...
+
+# Overrides spec.type of a specific Kubernetes Service
+# Equivalent helm cli flag --set swaggerui.portType=LoadBalancer
+swaggerui:
+-  portType: ClusterIP
++  portType: LoadBalancer
+
+# Overrides spec.type of all Open Match components
```

Comment:
Are these values shared across subcharts? So which all components does this service portType impact? All across all core charts and subcharts? 

Also, does synchronizer use this? if not, might be best to call out which components can be configured using this.